content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -47,6 +61,23 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 			return FileDataElementWrapper.getFromElement(this._domElement);
 		},
 
+		/**
+		 * Initializes object fields.
+		 */
+		_init : function() {
+
+			// Retrive file name
+			var fileInfoDom = this._domElement.getElementsByClassName(
+				FileElementWrapper.CLASS_FILE_INFO);
+
+			var fileTextSpan = fileInfoDom[0].getElementsByClassName(
+				FileElementWrapper.CLASS_SELECTABLE_TEXT);
+
+			var nameSrcDom = fileTextSpan.length === 0 ? fileInfoDom[0] : fileTextSpan[0];
+			this._fileName = nameSrcDom.getAttribute("title");
```

Comment:
Mine and your's mistake. I did not investigate the problem enough due to slow internet connection, so I just solved it with this approach. The problem was that some divs had the different structure. Now, I've dig into the problem and found out the cause. Commit page helper will in method getAllFiles return all divs/spans which have class 'file-info'. The problem is that commented files will be represented under **Conversation** tab of this page as well, with 'file-info' class. So we have to fix that method.